packages/pass-style/src/iter-helpers.js

```diff
@@ -12,8 +12,14 @@ import { Far } from './make-far.js';
  * @returns {Iterable<U>}
  */
 export const mapIterable = (baseIterable, func) =>
-  /** @type {Iterable<U>} */
-  Far('mapped iterable', {
+  /**
+   * An `Iterable` should not be a `Remotable` because it has a symbol-named
+   * method. That's why we use `harden` rather than, say, `Far`. However,
+   * an `Iterator` can be a `Remotable`.
+   *
+   * @type {Iterable<U>}
+   */
+  harden({
```

Comment:
This is used by code in agoric-sdk, but it doesn't seem like anything relies on these Iterables being remotable. 

TODO test in agoric-sdk.